### PR TITLE
feat: automerge renovate non-major updates if ci passes

### PR DESCRIPTION
**Pull request by c0state**
We basically rubber stamp non-major updates if CI passes so I think we can have renovate auto-merge said PRs

### CODE REVIEW

File: default.json [hunk 1]
```diff
@@ -9,5 +11,14 @@
       }
     }
   ],
-  "npmrc": "@quantumsi:registry=https://npm.pkg.github.com/"
+  "npmrc": "@quantumsi:registry=https://npm.pkg.github.com/",
+  "packageRules": [
+    {
+      "updateTypes": [
```
Comment (c0state) [inline default.json:17]: This should be additive with the upstream 4c shared config https://github.com/4Catalyzer/renovate-config, but unsure. A bit hard to test...

Comment (matt-mclaughlin-quantum-si) [reply default.json:17]: @c0state Looking at dependabot currently, is this behavior something we'd want to replicate there?

**Review by matt-mclaughlin-quantum-si — APPROVED** [review]
@c0state looks good - but it'll be short-lived I think because of the dependabot rollout

**Comment by c0state** [comment]
Closing since we're ready to move to Dependabot

**Pull request closed without merging**